test.py

```diff
@@ -23,32 +24,18 @@ def levenshtein_distance(s1, s2):
-        ]
-        self.expected_text = [
-            'Федеральное', 'государственное', 'бюджетное', 'образовательное', 'учреждение', 'высшего', 'образования',
-            'РОССИЙСКАЯ', 'АКАДЕМИЯ', 'НАРОДНОГО', 'ХОЗЯЙСТВА', 'и', 'ГОСУДАРСТВЕННОЙ', 'СЛУЖБЫ', 'при', 'ПРЕЗИДЕНТЕ',
-            'РОССИЙСКОЙ', 'ФЕДЕРАЦИИ', 'Институт', 'государственной', 'службы', 'и', 'управления', 'Направление',
-            'подготовки:', '38.04.01', 'Экономика', ' ', 'Образовательная', 'программа:', '_', 'Государственное',
-            'регулирование', 'экономики', ' ', 'МАГИСТЕРСКАЯ', 'ДИССЕРТАЦИЯ', 'ГОСУДАРСТВЕННО-ЧАСТНОЕ', 'ПАРТНЕРСТВО',
-            'И', 'ЕГО', 'РОЛЬ', 'В', 'АКТИВИЗАЦИИ', ' ', 'И', 'РАЗВИТИИ', 'ИНВЕСТИЦИОННОЙ', 'ДЕЯТЕЛЬНОСТИ', 'НА',
-            'ТЕРРИТОРИИ', ' ', 'РЕГИОНА', ' ', 'Автор:', 'обучающийся', 'группы', '3М18-34', 'заочной', 'формы',
-            'обучения', ' ', '/', 'Жилов', 'Ахмед-Хан', '/', 'Темурович', 'Руководитель:', 'Доцент', 'кафедры',
-            'государственного', 'регулирования', 'экономики,', 'кандидат', 'экономических', 'наук', '/', '_Шетов',
-            'Артур', 'Арсенович', '/', ' ', 'Москва,', '2021', 'г.'
-        ]
+        self.expected_len = 85
+        self.expected_text = "Федеральное государственное бюджетное образовательное учреждение высшего образования РОССИЙСКАЯ АКАДЕМИЯ НАРОДНОГО ХОЗЯЙСТВА и ГОСУДАРСТВЕННОЙ СЛУЖБЫ при ПРЕЗИДЕНТЕ РОССИЙСКОЙ ФЕДЕРАЦИИ Институт государственной службы и управления Направление подготовки: 38.04.01 Экономика   Образовательная программа: _ Государственное регулирование экономики   МАГИСТЕРСКАЯ ДИССЕРТАЦИЯ ГОСУДАРСТВЕННО-ЧАСТНОЕ ПАРТНЕРСТВО И ЕГО РОЛЬ В АКТИВИЗАЦИИ   И РАЗВИТИИ ИНВЕСТИЦИОННОЙ ДЕЯТЕЛЬНОСТИ НА ТЕРРИТОРИИ   РЕГИОНА   Автор: обучающийся группы 3М18-34 заочной формы обучения   / Жилов Ахмед-Хан / Темурович Руководитель: Доцент кафедры государственного регулирования экономики, кандидат экономических наук / _Шетов Артур Арсенович /   Москва, 2021 г."
         self.bboxes, self.text = get_bboxes()
 
     def test_bboxes_count(self):
-        self.assertEqual(len(self.bboxes), len(self.expected_bboxes),
-                         f"Количество BBox'ов ({len(self.bboxes)}) не совпадает с ожидаемым ({len(self.expected_bboxes)}).")
+        self.assertEqual(len(self.bboxes), self.expected_len,
+                         f"Количество BBox'ов ({len(self.bboxes)}) не совпадает с ожидаемым ({self.expected_len}).")
 
     def test_levenshtein_distance(self):
         result_text = ' '.join(self.text)
-        expected_text = ' '.join(self.expected_text)
-        distance = levenshtein_distance(result_text, expected_text)
-        self.assertLessEqual(distance, 0,
+        distance = levenshtein_distance(result_text, self.expected_text)/len(result_text)*100
```

Comment:
change distance to levenshtein_distance/len(result_text)
and change assert on assertLessEqual(distance, 0.05, ...)